src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  font-size: 14px;
+  text-decoration: none;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+$main-color: #060b35;
+$btn-color: #00acdc;
+$white-bg-color: #fff;
+
+.card {
+  width: 200px;
+  height: 406px;
+  background-color: $white-bg-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 32px 16px 16px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: #616070;
+  }
+
+  &__evaluetion {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__stars {
+    display: flex;
+    gap: 4px;
+  }
+  &__reviews {
+    margin: 0;
+    padding: 0;
```

Comment:
I've noticed that there is a lot of such resets. Please provide it upper in format (you can use more tags for such reset):
`
img,
h3,
a,
p {
  margin: 0;
  padding: 0;
}
`